plugins/com.python.pydev.analysis/src/com/python/pydev/analysis/refactoring/quick_fixes/AddTokenAndImportStatement.java

```diff
@@ -247,6 +253,47 @@ public void createTextEdit(ComputedInfo computedInfo) {
         }
     }
 
+    // function to get the amount of commas at the end of import string
+    private int verifyComma(String line) {
+        // TODO Auto-generated method stub
+        boolean withComma = true;
+        int commaAmount = 0;
+        int len = line.length();
```

Comment:
Why are you keeping the `len` here? It becomes unsynchronized when you do the `.trim()` later on, but I think you don't need to keep this variable at all (just use `line.length()` when you need it).